README.md

```diff
@@ -73,13 +73,13 @@ const logger = require('debugnyan')('foo', {}, { suffix: 'module', simple: false
 ## Tests
 
 ```
-❯ npm test
+❯ yarn test
 ```
 
 ## Release
 
 ```sh
-❯ npm version [<new version> | major | minor | patch] -m "Release %s"
+❯ yarn version [<new version> | major | minor | patch]
```

Comment:
Shouldn't this be `yarn release` instead of `yarn version`?

Also, I think this would be clearer / more explicit:

```
❯ yarn release ["major" | "minor" | "patch" | <custom version number>] # default: patch
```

Reply:
You're right, it's `release`, not `version`, didn't even notice that.

About the rest of the text, I don't have a strong opinion, I was just recommending the generic text we're using on other projects.

Reply:
I know. 🙂 I just wanted to take the opportunity to start migrating towards a clearer phrasing (the usual one was confusing to me the first time I saw it).

Reply:
So, are we aiming for this one as standard? 

`❯ yarn release ["major" | "minor" | "patch" | <custom version number>] # default: patch
`

Reply:
I'd at least remove the quotes there, since they might suggest the command is to be run with them.

Reply:
Using quotes has no disadvantage IIUC (the command works as expected), and explicitly indicates that these are specific values to be inputted as-is. So I see no downsides and a slight upside to keeping the quotes.

That said, it's not a strong preference, so if it bothers you a lot, we can remove them.